tests/api-compatibility/acceptable-breakages-v13.0.99.txt

```diff
@@ -0,0 +1,15 @@
+Compat issues with assembly Mono.Android:
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.Surface, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.Surface, Android.Graphics.Rect, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.SurfaceView, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.SurfaceView, Android.Graphics.Rect, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.Window, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'public void Android.Views.PixelCopy.Request(Android.Views.Window, Android.Graphics.Rect, Android.Graphics.Bitmap, Android.Views.PixelCopy.IOnPixelCopyFinishedListener, Android.OS.Handler)' does not exist in the implementation but it does exist in the contract.
+CannotRemoveAttribute : Attribute 'System.ObsoleteAttribute' exists on 'Dalvik.SystemInterop.DexFile' in the contract but not the implementation.
+CannotAddAbstractMembers : Member 'public Java.Nio.ByteBuffer Java.Nio.ByteBuffer.Slice(System.Int32, System.Int32)' is abstract in the implementation but is missing in the contract.
```

Comment:
This confuses me; we're using `compatVirtualMethod`, which should emit a *`virtual`* method, not an `abstract` method, so why is this warning emitted at all?

Reply:
This was a rebase conflict error that I fixed incorrectly.  Updated the `acceptable-breakages` to the only actual entry.